juneau-core/juneau-marshall/src/main/java/org/apache/juneau/internal/IOUtils.java

```diff
@@ -578,8 +570,6 @@ public static void close(Object...o) throws IOException {
 		IOException ex = null;
 		for (Object o2 : o) {
 			try {
-				if (o2 instanceof InputStream)
-					((InputStream)o2).close();

```

Comment:
The IOUtils.close(Object...) method is also a general utility method that's supposed to work on InputStreams.  So this code also should not be removed.

Reply:
As stated above, there doesn't seem to be any reason to call close on an InputStream.

Reply:
Done.